airflow/ti_deps/deps/trigger_rule_dep.py

```diff
@@ -146,6 +146,9 @@ def _evaluate_trigger_rule(
             elif tr == TR.ONE_FAILED:
                 if upstream_done and not (failed or upstream_failed):
                     ti.set_state(State.SKIPPED, session)
+            elif tr == TR.ALL_DONE:
```

Comment:
This is a quick fix for ALL_DONE trigger rule fire when all upstream tasks are skipped.